editor/components/default-block-appender/style.scss

```diff
@@ -1,21 +1,37 @@
 $empty-paragraph-height: $text-editor-font-size * 4;
 
+.editor-default-block-appender {
```

Comment:
I think those styles should be moved out of the `editor/components` directory. Because they only make sense in the `edit-post` layout (not in p2 for example). In `editor/components` we should think of the components (and styles) as reusable components that should look good when inserting them but if we want to position them properly etc... it should be done in the "layout" IMO.

In general, I do this in the. `modes/visual-editor/style.scss` by specifying the parent `.editor-visual-editor  +.editor-default-block-appender { `